Deployment/Testing/users.yml

```diff
@@ -166,6 +166,8 @@ data:
         flow_tenant_form_parameter: tenant
       consent:
         flow_id_query_parameter: consent_challenge
+        tenants:
+          tenant-a: Cool Company AS
```

Comment:
Seems like this is in the wrong place here? Would also be nice to have the default value (empty map) set in the `Source/Login/Backend/.docker/config.yaml` file - so that it can be used as a reference for what can be set.

Reply:
Correct - it was in the wrong place 😄